src/gromacs/nblib/molecules.cpp

```diff
@@ -206,8 +257,8 @@ std::vector<std::tuple<int, int>> Molecule::getExclusions() const
 
         int secondIndex = std::get<2>(*it2);
 
-        ret.emplace_back(std::make_tuple(firstIndex, secondIndex));
-        ret.emplace_back(std::make_tuple(secondIndex, firstIndex));
+        ret.emplace_back(firstIndex, secondIndex);
```

Comment:
Does this not need to be a tuple anymore?

Reply:
`emplace_back` also accepts the parameters for the class constructor that you are passing.

Reply:
TIL something new!